.github/searchForm.js

```diff
@@ -0,0 +1,17 @@
+class SearchForm {
```

Comment:
this class should contain all the logic that relates to the form - creating the form and rendering it, fetching the data from the server, debouncing etc...